### PR TITLE
Initial code profiling

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**

### CODE REVIEW

**Comment by abarciauskas-bgse** [comment]
@vincentsarago could you take a look at the error here: https://github.com/developmentseed/tile-benchmarking/blob/9b259ce6c3b3efb0814d327365e821d17f7f2c1d/profiling/profile.ipynb

I started to try and debug it but you probably know the answer right away. Will investigate more tomorrow if you don't get to it first :)

File: profiling/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+geojson_pydantic
```
Comment (abarciauskas-bgse) [inline profiling/requirements.txt:1]: should add specific versions

**Comment by abarciauskas-bgse** [comment]
@vincentsarago @sharkinsspatial there are still a few small things to clean up but this is ready for your review.

**Comment by vincentsarago** [comment]
@abarciauskas-bgse can you make sure you have those env set:
      - GDAL_INGESTED_BYTES_AT_OPEN=32768
      - GDAL_DISABLE_READDIR_ON_OPEN=EMPTY_DIR
      - GDAL_HTTP_MERGE_CONSECUTIVE_RANGES=YES
      - GDAL_HTTP_MULTIPLEX=YES
      - GDAL_HTTP_VERSION=2
      - VSI_CACHE=TRUE

**Comment by abarciauskas-bgse** [comment]
Thanks @vincentsarago I will check out those env variables. Also will look into the potential to do this in a git runner, instead of my mac. Just thinking about if I should work on setting up a git runner or running on JupyterHub. Do you have an opinion on that

**Comment by vincentsarago** [comment]
@abarciauskas-bgse I'm not sure what is the best TBH. Ideally we would run benchmark outside the notebook environment using a `simple` python script but I don't know how much of a refactor it will mean

**Comment by abarciauskas-bgse** [comment]
@vincentsarago you are correct, adding those env variables speeds things up a lot!

I have added them to the notebook and am closing this PR in favor of https://github.com/developmentseed/tile-benchmarking/pull/15 which has those environment variables and pinned python library versions

**Pull request closed without merging**